Source/WebKit/Shared/API/Cocoa/WKRemoteObjectCoder.mm

```diff
@@ -799,6 +799,8 @@ static id decodeObjectFromObjectStream(WKRemoteObjectDecoder *decoder, const Has
         (__bridge CFTypeRef)NSUUID.class,
         (__bridge CFTypeRef)NSError.class,
         (__bridge CFTypeRef)NSURLError.class,
+        (__bridge CFTypeRef)NSDate.class,
+        (__bridge CFTypeRef)NSClassFromString(@"NSDecimalNumberPlaceholder"),
```

Comment:
Do you need to add `NSDecimalNumber` explicitly as well?

Reply:
Apparently not, but there's no reason not to add it while allowing its subclass.